setup.py

```diff
@@ -1,6 +1,8 @@
 from pkgutil import iter_modules
 
 # To be replaced by: from setuptools_scm import get_version
+
+
```

Comment:
My auto-formatter wanted this. Debatable.

Reply:
Agree, remove it.